docker-compose.yml

```
@@ -161,6 +162,28 @@ services:
       args:
         BASE_IMAGE: 'sentry-onpremise-local'
     command: '"0 0 * * * gosu sentry sentry cleanup --days $SENTRY_EVENT_RETENTION_DAYS"'
+  nginx:
+    ports:
+      - '9000:80/tcp'
+    image: "nginx:1.16.1"
```

Should we go with `1.16` so we'd get patch releases automatically? Also the latest version seems to be `1.17`.

---

I don't have an opinion on this one, so it is your call. 
I was told to 'fix' the image versions for external dependencies so we get consistent/repeatable environments.

---

We are okay fixing the images to minor versions which we do with `postgres` and `redis` already so let's change this to `1.16` to be consistent with others.
```suggestion
    image: "nginx:1.16"
```

---

ok